libz-rs-sys-cdylib/zpipe.c

```diff
@@ -42,8 +52,14 @@ int def(FILE *source, FILE *dest, int level)
     unsigned char out[CHUNK];
 
     /* allocate deflate state */
-    strm.zalloc = Z_NULL;
-    strm.zfree = Z_NULL;
+#ifdef NO_STD
+    strm.zalloc = custom_zalloc;
+    strm.zfree = custom_zfree;
+#else
+    strm.zalloc = NULL;
+    strm.zfree = NULL;
+#endif
```

Comment:
Unconditionally using custom_zalloc/custom_zfree would be fine too, right?

Reply:
it would work, but we use this test for running the various allocators in CI